.github/workflows/go-ci.yml

```diff
@@ -47,6 +47,24 @@ jobs:
           version: v1.59
           skip-cache: true
 
+  lint-imports:
+    needs: [setup]
+    name: Lint Imports
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-go@v5
+        with:
+          go-version: ${{ inputs.go-version }}
+
+      - name: install goimports-reviser
+        run: go install -v github.com/incu6us/goimports-reviser/v3@latest
```

Comment:
Why this linter? Are we using it at all?

Reply:
Also, I see install but no start.

Reply:
>Why this linter? Are we using it at all?

It is a linter, that checks the `make sort-imports` command has been ran on the repo. And util is a part of `make lint` command, that is used to check lint rules locally. 

> Also, I see install but no start.

It is run in the following command: `make lint-imports`

Reply:
For the context: we use this util, because 
- golangci-lint imports order linter does not have `fix` option to quickly order of the imports the way we like. This util provides both `linter` and `fmt`
- it is a bit stricker than golangci-lint linter and groups imports into 4 groups: 
    - std
    - local repo
    -  same org
    -  external orgs

Reply:
>It is run in the following command: make lint-imports

```
lint: lint-imports
	@echo "--> Running linter"
	@golangci-lint run
	@markdownlint --config .markdownlint.yaml '**/*.md'
	@cfmt --m=120 ./...
.PHONY: lint
```

huh?

Reply:
First line embeds `lint-imports`:
```
lint: lint-imports
```

Reply:
Oh, thanks.